impl/src/main/java/dao/Vote.java

```diff
@@ -0,0 +1,48 @@
+package dao;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
+public class Vote {
+
+    long userId;
+
+    long answerId;
+
+    @Max(1)
+    @Min(-1)
+    int value;
+
+    public Vote() {
```

Comment:
can't remember, but is there a need or the public constructor?

Reply:
Not more than anywhere, as far as I can see. :P